packages/ingest/queue.js

```diff
@@ -102,11 +92,11 @@ async function enqueueGranuleIngestMessage(
   if (parentExecutionArn) message.cumulus_meta.parentExecutionArn = parentExecutionArn;
 
   message.cumulus_meta.execution_name = uuidv4();
-  const arn = getExecutionArn(
+  const arn = aws.getExecutionArn(
     message.cumulus_meta.state_machine,
     message.cumulus_meta.execution_name
   );
-  await sendSQSMessage(queueUrl, message);
+  await aws.sendSQSMessage(queueUrl, message);
```

Comment:
Changing these aws package helpers to be used off of the `aws` object makes it possible to `sinon.stub` this function and inspect its arguments, which is necessary to ensure that the granule objects don't get transformed